docs/chapter_array_and_linkedlist/array.md

```diff
@@ -105,6 +105,25 @@
     int nums[5] = { 1, 3, 2, 5, 4 };
     ```
 
+=== "Kotlin"
+
+    ```kotlin title="array.kt"
+    // 初始化数组
+    val arr = arrayOf(1, 2, 3, 4, 5)
```

Comment:
The Kotlin code is not consistent with other languages. Please fix it referring to Python or Java code.